### DISCUSSION TITLE
How to open links in new tab

### DISCUSSION THREAD

**Discussion by WilliamEstrela** [requarks/wiki#2881] [Help / Questions]
How to add a function to open a link in new tab ?

**Comment by scienceasdf** [comment]
Login as administrator, go to **administrator dashboard** -> **themes**, then insert the following code to the head:
```
<base target="_blank">
```

Save this modify, and you will open every new hyperlink in a new tab.

**Reply by okoestner** [reply]
@scienceasdf  Is there a chance to only open external links in a new tab? And keep all Wiki links in the same one?
